lib/process-services/process-list/components/start-process.component.ts

```diff
@@ -96,37 +96,41 @@ export class StartProcessInstanceComponent implements OnChanges {
             this.appId = changes['appId'].currentValue;
         }
 
+        if (changes['processDefinitionId'] && changes['processDefinitionId'].currentValue) {
+            this.processDefinitionId = changes['processDefinitionId'].currentValue;
+        }
+
         this.loadStartProcess();
     }
 
     public loadStartProcess() {
         this.resetSelectedProcessDefinition();
         this.resetErrorMessage();
 
-        if (this.appId) {
-            this.activitiProcess.getProcessDefinitions(this.appId).subscribe(
-                (processDefinitionRepresentations: ProcessDefinitionRepresentation[]) => {
-                    this.processDefinitions = processDefinitionRepresentations;
-
-                    if (this.processDefinitions.length === 1) {
-                        this.currentProcessDef = JSON.parse(JSON.stringify(this.processDefinitions[0]));
-                    } else {
-                        if (this.processDefinitionId) {
-                            this.processDefinitions = this.processDefinitions.filter((currentProcessDefinition) => {
-                                return currentProcessDefinition.id === this.processDefinitionId;
-                            });
-                            this.currentProcessDef = JSON.parse(JSON.stringify(this.processDefinitions[0]));
-                        }
+        this.activitiProcess.getProcessDefinitions(this.appId).subscribe(
+            (processDefinitionRepresentations: ProcessDefinitionRepresentation[]) => {
+                this.processDefinitions = processDefinitionRepresentations;
+
+                if (this.processDefinitions.length === 1 || !this.processDefinitionId) {
```

Comment:
what happens if I have multiple process definitions and I didn't use the processDefId ?
This condition will be 
false || true